src/components/FormSections/PersonalInfo.jsx

```diff
@@ -189,7 +190,7 @@ function PersonalInfo(props){
                         >
                             <div data-testid="formGroup" className="usa-form-group usa-form-group--month">
                                 <label data-testid="label" className="usa-label" htmlFor="date_of_birth_month">
-                                    Month
+                                <span dangerouslySetInnerHTML= {{__html: string.month}}/>
```

Comment:
we do not need the dangerouslysetinnerhtml
set to `{string.month}`

Reply:
removed the dangerouslysetinnerhtml and fixed format